partner_communication_switzerland/models/mail_tracking_event.py

```diff
@@ -61,10 +61,16 @@ def process_reject(self, tracking_email, metadata):
         if metadata.get('error_type') == 'Invalid' and 'RBL' not in \
                 metadata.get('error_description', '') and not partner.user_ids:
             self._invalid_email(tracking_email)
-            partner.write({
+            to_write = {
                 'invalid_mail': partner.email,
                 'email': False
-            })
+            }
+            if partner.email_only:
+                del to_write['email']
+                partner.message_post('Warning : Email invalid but sponsor '
```

Comment:
Add parameters to notify someone of the staff. Like you did now, it will post the message but nobody will be notified about this. Take a look at an [example here](https://github.com/CompassionCH/compassion-switzerland/blob/10.0/wordpress_connector/models/contracts.py#L151).
You can add another staff.notification.settings for this (maybe one for general notifications).